build-ps/rpm/mysqld.cnf

```diff
@@ -9,9 +9,49 @@ datadir=/var/lib/mysql
 socket=/var/lib/mysql/mysql.sock
```

Comment:
@vadimtk as I remember all changes to my.cnf should be approved by you. so please take a look at these changes.
We are going to change the way how we provide my.cnf and follow PS way